### DISCUSSION TITLE
Docusaurus v2.3 and Docusaurus OpenAPI Docs

### DISCUSSION THREAD

**Discussion by gii-pan** [PaloAltoNetworks/docusaurus-openapi-docs#408] [Q&A] [answered]
Hello @sserrata , 

I'd like to know if in the next release are you planning to update the plugin to the new version of Docusaurus(v2.3)?

Do you have a deadline?

Thank you 🥰

**Comment by sserrata** [comment, marked as answer]
Hi @gii-pan, although we are planning to support v2.3.x we would prefer to do so without breaking backward compatibility. That said, it might be difficult given the breaking changes introduced to the Docusaurus `Tabs` component. Worst case, we'll move forward with supporting 2.3.x+ and will document a compatibility matrix for users who aren't ready to upgrade yet.

**Comment by sserrata** [comment]
Support for 2.3.0 is now available in v2.0.0-beta.1
